scripts/Diffraction/isis_powder/routines/common_output.py

```diff
@@ -8,6 +8,42 @@
 import os
 
 
+def split_into_tof_d_spacing_groups(run_details, processed_spectra):
```

Comment:
Thanks for making those changes. We've got a fair bit of repetition here though. Could we add a boolean flag like `include_qsquared` to this instead? and then just have 1 function but the return either returns 2 or 3 things?